chapters/functions.tex

```diff
@@ -1880,10 +1880,11 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 "GenerateEvents" "=" ( false | true )
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-Has only an effect within a function declaration
-
-If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events (one possibility of doing this is to inline the function and generate events for the inlined function).
+Has only an effect within a function declaration.
 
+If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events.
+As defaults this implies \lstinline!Inline = true!, but it can be overriden by specifying one of the inline-annotations (with true value).
+As event-generation can be crucial for correct simulation, a diagnostic is strongly recommended in case events will not be generated for a function with `GenerateEvents = true`.
```

Comment:
As noted by @eshmoylova and @casella (and I totally agree), it would be great if we could simply drop this line and instead make `GenerateEvents` imperative.

Reply:
Ok

Reply:
I think that is a good to make it clear that GenerateEvents shall truly generate events.

Reply:
As the description of `GenerateEvents = true` now says _shall_, this sentence should be removed, and then we can resolve this conversation as long as we remember to bring up the change from _proposes_ to _shall_ for a language group decision.

Reply:
Ok